docs/schema.md

```diff
@@ -13,15 +13,15 @@ The primary key in Scalar DB is composed of one or more partition keys and zero
 Data types supported in Scalar DB are a little more restricted than in Cassandra.
 Here are the supported data types and their mapping to Cassandra data types.
 
-|Scalar DB  |Cassandra  |
-|---|---|
-|BOOLEAN  |boolean  |
-|INT  |int  |
-|BIGINT  |bigint  |
-|FLOAT  |float  |
-|DOUBLE  |double  |
-|TEXT  |text  |
-|BLOB  |blob  |
+|Scalar DB  |Cassandra  |Cosmos DB  |DynamoDB  |MySQL  |PostgreSQL |Oracle |SQL Server |
+|---|---|---|---|---|---|---|---|
+|BOOLEAN  |boolean  |boolean   |boolean     |boolean    |boolean    |number(1) |bit  |
+|INT  |int  |int    |ScalarAttributeType/N    |int    |int    |int    |int    |int    |
+|BIGINT  |bigint    |bigint |ScalarAttributeType/N |bigint |bigint |number(19) |bigint | 
```

Comment:
For Cosmos DB, is it correct? Where did you get it from?

Reply:
@feeblefakie san,
Cosmos DB supports all the data types that are supported by Cassandra. I got it from the following documentation
https://docs.microsoft.com/en-us/azure/cosmos-db/cassandra-support

Reply:
That is not the right documentation to see.
We don't use Cassandra API of Cosmos DB. We write an adapter by ourselves, thus, we map data types by ourselves.